reference-data/ill-policies/WillNotLend.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "b0f97013-87f5-4bab-87f2-ac4a5191b489",
-  "name": "will not lend",
+  "name": "Will not lend",
```

Comment:
Does this need to change in existing environments as well? If so, then there needs to be an upgrade script as well.

Please check with Charlotte

Reply:
Yes, I found in an issue comment that an upgrade script has been requested. I'm working on it now.

Reply:
I added an upgrade script and I think I've triggered it properly.